app/controllers/application_controller.rb

```diff
@@ -88,6 +88,19 @@ def require_logged_in_moderator
     end
   end
 
+  def require_logged_in_admin
+    require_logged_in_user
+
+    if @user
```

Comment:
`unless @user.try(:is_admin?)`

Reply:
Tried that, it gives a double render error if you try to access one of the protected pages when there is no user logged in. As long as we're using `require_logged_in_user` as-is, it needs to only render when `@user` is not `nil` and also not admin. I agree that this is pretty awkward, but I think it needs a broader refactoring of how authorization is done that seems outside the scope of this PR.